modules/govuk/manifests/apps/collections.pp

```diff
@@ -31,6 +31,15 @@
 # [*memcache_servers*]
 #   URL of a shared memcache cluster.
 #
+# [*google_tag_manager_id*]
+#   The ID for the Google Tag Manager account
+#
+# [*google_tag_manager_preview*]
+#   Allows a tag to be previewed in the Google Tag Manager interface
+#
+# [*google_tag_manager_auth*]
+#   The identifier of an environment for Google Tag Manager
+#
```

Comment:
I wonder if `google_tag_manager_env_id` would be more descriptive for this?

Reply:
Yeah it probably is, but `google_tag_manager_auth` is already used by content-data-admin and content-publisher (two things I can't change) so I think I'd rather stick with consistency for the moment.

Reply:
Fair point - at least it's consistently a bad name 😉

Reply:
Thanks. These should be temporary anyway while we test this stuff, it'll be removed before go live.